### ISSUE TITLE
[TF2] Antarctic Eyewear on Scout LOD issue

### ISSUE THREAD

**Issue by leo-proct** [ValveSoftware/Source-1-Games#6188]
Currently the Antarctic Eyewear on Scout has a issue where the texture on the goggles seems to disappear on lower LOD making it appear white

Example.
![image](https://github.com/user-attachments/assets/fc0cffb9-7e5a-466a-b16c-536654524238)

Current fix of model provided by Hellview24 on game banana
https://gamebanana.com/mods/201279 

Please consider updating the model with the fixed one on the site or revert the model pre May 17, 2018 patch

**Comment by CheddarSniffer** [comment]
If you haven't already, contact the creators on steam

**Comment by leo-proct** [comment]
> If you haven't already, contact the creators on steam

Yeah this has been raised, it seems to be an issue with UV map that got updated in a later patch post blue moon
